easyrest/views/restaurant_controler.py

```diff
@@ -0,0 +1,49 @@
+from pyramid.view import view_config
+from pyramid.response import Response
+
+from pyramid.httpexceptions import HTTPNotFound
+
+from sqlalchemy.exc import DBAPIError
+
+from json import dumps
+
+from ..models.Restaurant import Restaurant
+
+
+def asign_tags(rests):
+    rests_dict = [rest.as_dict() for rest in rests]
+    for i, rest in enumerate(rests):
```

Comment:
You don't need `enumerate` dict is mutable type so it's returned by pointer and can be updated in-place.
```
for restaurant in restaurants:
    del restaurant["menu_id"]
    restaurant["id"] = ...
```